data/json/itemgroups/guns.json

```diff
@@ -159,6 +159,7 @@
       [ "fn_fal", 40],
       [ "hk_g3", 40 ],
       [ "hk_g36", 30 ],
+      [ "henry_big_boy", 30],
```

Comment:
[https://www.henryrifles.com/rifles/big-boy/](url)
Not as much as you might think. I may have misstated what this rifle is supposed to be in the OP. Its a modern production lever action rifle for cowboy action shooting hobbyists.